internal/services/mongocluster/mongo_cluster_resource_test.go

```diff
@@ -92,7 +92,29 @@ func TestAccMongoCluster_previewFeature(t *testing.T) {
 		},
 		data.ImportStep("administrator_password", "create_mode"),
 		{
-			Config: r.geoReplica(data),
+			Config: r.geoReplica(data, r.previewFeature(data)),
```

Comment:
In the acceptance test we're still setting `preview_features = ["GeoReplicas"]` in the config but in the notes it says this isn't required anymore?

Reply:
Although `GeoReplicas` is now generally available, the current API behavior is still supported for `preview_features = ["GeoReplicas"]`. Therefore, I still keep the test case for `preview_features` property. Do you mean I should remove that test case? Any feedback feel free to let me know, thank you.